ss58-registry.json

```diff
@@ -201,7 +201,7 @@
     },
     {
       "prefix": 21,
-      "network": "dock-testnet",
+      "network": "dock-pos-testnet",
```

Comment:
Is it a canary/incentivised network or testnet?

Reply:
just a testnet

Reply:
Testnets don’t need their own prefix, as it is not a technical constrain at all

Reply:
have updated to remove the testnet entry altogether